src/ophyd_async/panda/panda_utils.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Optional
+
+from ophyd_async.core import (
+    get_signal_values,
+    load_from_yaml,
+    save_to_yaml,
+    set_signal_values,
+    walk_rw_signals,
+)
+from ophyd_async.panda import PandA
+
+
+def _get_panda_phases(panda: PandA, ignore: Optional[List[str]] = None):
+    # Panda has two load phases. If the signal name ends in the string "UNITS", it needs to be loaded first so put in first phase
+    signalRW_and_value = yield from get_signal_values(walk_rw_signals(panda), ignore)
+    phase_1 = {}
+    phase_2 = {}
+    for signal_name in signalRW_and_value.keys():
+        if signal_name[-5:] == "units":
+            phase_1[signal_name] = signalRW_and_value[signal_name]
+        else:
+            phase_2[signal_name] = signalRW_and_value[signal_name]
+
+    return [phase_1, phase_2]
```

Comment:
I already wrote this yesterday for i22, so I'll paste in what I had in case you want to use any of it:
```python
def save_the_panda():
    signals = walk_rw_signals(panda)
    values = yield from get_signal_values(signals, ignore=["seq1.table"])
    units = {n: values.pop(n) for n in list(values) if n.endswith("units")}
    save_to_yaml([units, values], "/tmp/thing.yaml")
```